crates/router/src/connector/utils.rs

```diff
@@ -76,9 +76,11 @@ pub trait RouterData {
 }
 
 pub fn get_unimplemented_payment_method_error_message(connector: &str) -> String {
-    format!("Selected payment method through {}", connector)
+    format!("{} through {}", SELECTED_PAYMENT_METHOD, connector)
```

Comment:
This could have been:

```suggestion
    format!("{SELECTED_PAYMENT_METHOD} through {connector}")
```

Optional change, btw.